docs/dev_coding_standards.rst

```diff
@@ -0,0 +1,41 @@
+.. _dev_coding_standards:
```

Comment:
For me it is a single standard so the plurality here doesn't hold.

Reply:
Well, you've said that we use google plus some tweaks ... so that makes more than one ;) And just as a title, "standard" doesn't really read well.